src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
Added a benchmark. It starts from an empty action list and adds actions one by one. 

Before: `29.16 ops/sec ±25.13% (18 runs sampled)`.
After adding `react-dnd`'s `DragDropContext(HTML5Backend)` and `DragSource` hocs:   `25.05 ops/sec ±25.87% (18 runs sampled)`.

When starting not from an empty list, but having already 500 actions, it's `4.98 ops/sec ±4.29%` (without hocs) and `3.64 ops/sec ±4.39%` (with `react-dnd`). But we're usually avoiding such bad perf with `maxAge`. Also note that I'm benchmarking production build, since it's not recommended to benchmark React in development and we're not using it for the extension. Without the extension we're running on dev environment and the perf should be worse.

Obviously, Dragula has no impact, as nothing is done on component update. Unfortunately, I wasn't able to get `react-sortable-hoc` working on `jsdom` (it fails after calling `findDOMNode` inside the lib), but it shouldn't be much different from `react-dnd`, as there's lot of work inside the hocs on every update.

If the perf waste would be for a core feature, it could be negotiable, but not for this one. It's not something we'd use frequently, but nice to have (I just got inspired by [Dan's workflow](https://twitter.com/dan_abramov/status/761552171045359616)). We would also add the ability to duplicate actions (when dragging an action holding `Alt` key), which should be more popular.

Feel free to make changes in the pr when you have some time. It's not in a hurry, we're [using a fork](https://github.com/zalmoxisus/remotedev-inspector-monitor/commits/remotedev-inspector) meanwhile.